src/user-guide/writing-workflows/configuration.rst

```diff
@@ -64,13 +64,13 @@ Cylc supports two `template processors`_ for use in the flow.cylc file:
 Syntax
 ^^^^^^
 
-The following defines legal :cylc:conf:`flow.cylc` syntax:
+:cylc:conf:`flow.cylc` syntax, in general terms:
 
-- **Items** are of the form ``item = value``.
+- **Config items** are of the form ``item = value``.
```

Comment:
Do we not want to call these "settings"?

Reply:
Yeah, can do.